src/main/java/org/broadinstitute/hellbender/cmdline/StandardArgumentDefinitions.java

```diff
@@ -33,6 +33,9 @@ private StandardArgumentDefinitions(){}
     public static final String ADD_OUTPUT_SAM_PROGRAM_RECORD = "addOutputSAMProgramRecord";
     public static final String ADD_OUTPUT_VCF_COMMANDLINE = "addOutputVCFCommandLine";
     public static final String SEQUENCE_DICTIONARY_NAME = "sequenceDictionary";
+    public static final String ANNOTATION_LONG_NAME = "annotation";
+    public static final String ANNOTATION_GROUP_LONG_NAME = "group";
```

Comment:
I think this should really be named `annotationGroup`, regardless of what GATK3.x had (though check the WDLs for usage of the old name)

Reply:
Doesn't seem to be in any WDLs here or in dsde-pipelines